src/status_im/contexts/wallet/send/input_amount/component_spec.cljs

```diff
@@ -148,7 +125,7 @@
                                      {:crypto-decimals          10
                                       :limit-crypto             1000
                                       :on-confirm               on-confirm
-                                      :initial-crypto-currency? false}])
```

Comment:
Previously incorrect test that worked only because `input-amount` didn't communicate this field to internal `token-input` and latest maintained own (out-of-sync) atom for the same data.